server/graphcool.yml

```diff
@@ -29,24 +36,23 @@ permissions:
 # rootTokens:
 #   - mytoken
 
-
 # added by email-password template: (please uncomment)
- functions:
-   signup:
-     type: resolver
-     schema: src/email-password/signup.graphql
-     handler:
-       code: src/email-password/signup.ts
- 
-   authenticate:
-     type: resolver
-     schema: src/email-password/authenticate.graphql
-     handler:
-       code: src/email-password/authenticate.ts
- 
-   loggedInUser:
-     type: resolver
-     schema: src/email-password/loggedInUser.graphql
-     handler:
-       code: src/email-password/loggedInUser.ts
 
+# functions:
+#   signup:
+#     type: resolver
+#     schema: src/email-password/signup.graphql
+#     handler:
+#       code: src/email-password/signup.ts
+
+#   authenticate:
+#     type: resolver
+#     schema: src/email-password/authenticate.graphql
+#     handler:
+#       code: src/email-password/authenticate.ts
+
+#   loggedInUser:
+#     type: resolver
+#     schema: src/email-password/loggedInUser.graphql
+#     handler:
+#       code: src/email-password/loggedInUser.ts
```

Comment:
if you comment this out your graphcool sing in and signup queries and mutations will not work.

Reply:
Uncommented